### PR TITLE
Update/Add fission-core & fission-all helm charts

### PR DESCRIPTION
**Pull request by life1347**
This PR aims to add/update helm charts `fission-core` and `fission-all` to help users to install fission without painful process. Notice that Kubernetes version must be 1.6 or above since the charts set up RBAC rules by default. (This PR is not ready to merge until the deployment of message queue trigger is commit.)

### CODE REVIEW

**Comment by baldwinSPC** [comment]
@life1347 is there any sense on when this pr will be merged? cc @rimusz

**Comment by life1347** [comment]
Hi @baldwinSPC,
I'll push a commit that integrates message queue trigger this weekend. And hopefully, this PR can be merged next week. If you don't need such feature, you can just pull it to your local branch and try it out :)

File: charts/fission-core/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+name: fission
+version: 0.1.0
+description: A Helm chart for Fission. Fission is a fast serverless framework for Kubernetes.
+keywords:
+- fission
+- serverless
+home: http://fission.io/
+maintainers:
```
Comment (soamvasani) [inline charts/fission-core/Chart.yaml]: Can you add your name and email to this list, and mine as well?

(And I'm not sure Sanket still wants to be contacted about it. @ssudake21, do you still want to be in the list of maintainers for this chart?)

File: charts/fission-all/values.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Kubernetes configuration
+## For minikube, set this to NodePort, elsewhere use LoadBalancer.
+serviceType: LoadBalancer
+
+## Fission image repostiroy
+image: fission/fission-bundle
+
+## Fission image version
+imageTag: alpha20170124
```
Comment (soamvasani) [inline charts/fission-all/values.yaml]: Can you update this to the latest release (currently that's `nightly20170705`)

File: charts/fission-core/templates/NOTES.txt [hunk 3]
```diff
@@ -0,0 +1,37 @@
+1. Install the client CLI.
+
+Mac:
+  $ curl http://fission.io/mac/fission > fission && chmod +x fission && sudo mv fission /usr/local/bin/
```
Comment (soamvasani) [inline charts/fission-core/templates/NOTES.txt]: We've switched to using github releases for the CLIs.

Can you update this to the command from INSTALL.md:
```
curl -Lo fission https://github.com/fission/fission/releases/download/nightly20170705/fission-cli-osx && chmod +x fission && sudo mv fission /usr/local/bin/
```

File: charts/fission-core/templates/NOTES.txt [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+Linux:
+  $ curl http://fission.io/linux/fission > fission && chmod +x fission && sudo mv fission /usr/local/bin/
```
Comment (soamvasani) [inline charts/fission-core/templates/NOTES.txt]: same comment, please update to the command from INSTALL.md

File: charts/fission-core/templates/deployment.yaml [hunk 5]
```diff
@@ -1,3 +1,77 @@
...
+subjects:
+- kind: ServiceAccount
+  name: fission-admin
+  namespace: fission
+userNames:
+- system:serviceaccount:fission:fission-admin
+
+{{ else }}
+
+# For minikube, GKE
```
Comment (soamvasani) [inline charts/fission-core/templates/deployment.yaml]: nitpicking here, but this section really applies to all k8s except openshift, right?

File: charts/README.md [hunk 6]
```diff
@@ -5,15 +5,24 @@
 
 ## Prerequisites
 
-- Kubernetes 1.4+ with Beta APIs enabled
+- Kubernetes 1.6+ with Beta APIs and RBAC enabled
```
Comment (soamvasani) [inline charts/README.md]: Do we need RBAC actually *enabled*?  It's enough for the cluster to be 1.6+, right?

File: charts/README.md [hunk 7]
```diff
@@ -5,15 +5,24 @@
...
 
 
+## Helm charts
+
+The following table lists two helm charts for Fission.
+
+| Parameter      | Description                            |
+| ---------------| ---------------------------------------|
+| `fission-core` | Try with essential features of Fission |
+| `fission-all`  | Enjoy all cool features of Fission     |
```
Comment (soamvasani) [inline charts/README.md]: Nice marketing but let's try to be more specific 😉 
How about:
```
fission-core: FaaS essentials, and triggers for HTTP, Timers and Kubernetes Watches
fission-all: Log aggregation with fluentd and InfluxDB; NATS for message queue triggers; Fission-UI 
```

**Comment by soamvasani** [comment]
@baldwinSPC Let's say, sometime in the next 1-2 weeks. Let me know if you were hoping for it to be sooner :)

Meanwhile if you have thoughts about the chart itself feel free to comment.

File: charts/fission-core/Chart.yaml (hunk 1 above)
Comment (ssudake21) [inline charts/fission-core/Chart.yaml]: @soamvasani  @life1347  You can remove my name from maintainers list. I do plan to contribute minor fixes though . 😉

**Pull request merged**